esp/scm/ws_dfu.ecm

```diff
@@ -861,6 +861,8 @@ ESPrequest DFUFileAccessV2Request
     string RequestId;                   // optional
     int ExpirySeconds(60);              // how long the retrieved meta info is valid for, to access physical files (in dafilesrv)
     bool ReturnTextResponse(false);     // Returns response in full not just in blob    
+    [min_ver("1.51")] int64 SessionId(0); // session id of this client request    
+    [min_ver("1.51")] int LockTimeoutMs(300000); // Max seconds to block waiting for lock (default = 5 mins)
```

Comment:
comment claims timeout is in seconds, but its effectively a millisec timeout

Reply:
comment should say milliseconds, will change